modules/dkan/dkan_topics/dkan_topics.features.field_instance.inc

```diff
@@ -113,9 +112,8 @@ function dkan_topics_field_default_field_instances() {
     'display' => array(
       'default' => array(
         'label' => 'hidden',
-        'module' => 'dkan_topics',
         'settings' => array(),
-        'type' => 'dkan_topic_formatter',
+        'type' => 'hidden',
```

Comment:
This leads to spacing being to great between the topic and the rest of the story. Before it looked like this:

![2016-10-13_17-15-19](https://cloud.githubusercontent.com/assets/309671/19367324/a38c348c-9168-11e6-8380-ef0a1b87556b.png)

now it's like this

![2016-10-13_17-15-45](https://cloud.githubusercontent.com/assets/309671/19367400/fa268b62-9168-11e6-8d75-99b55df444e8.png)

Please provide a fix in the theme/css